config/software/openresty.rb

```diff
@@ -56,7 +58,15 @@
     # However, they require libatomic-ops-dev and libaio
     #'--with-file-aio',
     #'--with-libatomic'
-  ].join(" "), env: env
+  ]
+
+  if ((version == "1.4.3.6") &&
```

Comment:
But you make a good point. We might have to update this to support future versions as well. Honestly, I'm getting to the point where I'd rather break people and force them to upgrade.